[2023-06-27T09:09:15.870+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_stage permission to 509
[2023-06-27T09:09:15.871+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_stage permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_stage'
[2023-06-27T09:09:15.904+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_stage permission to 509
[2023-06-27T09:09:15.906+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_stage permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_stage'
[2023-06-27T09:09:15.938+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.create_stage manual__2023-06-27T09:09:11.133710+00:00 [queued]>
[2023-06-27T09:09:15.949+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.create_stage manual__2023-06-27T09:09:11.133710+00:00 [queued]>
[2023-06-27T09:09:15.950+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-27T09:09:15.967+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_stage> on 2023-06-27 09:09:11.133710+00:00
[2023-06-27T09:09:15.973+0000] {standard_task_runner.py:57} INFO - Started process 12032 to run task
[2023-06-27T09:09:15.977+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Setup', 'create_stage', 'manual__2023-06-27T09:09:11.133710+00:00', '--job-id', '199', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp7ggfdevw']
[2023-06-27T09:09:15.980+0000] {standard_task_runner.py:85} INFO - Job 199: Subtask create_stage
[2023-06-27T09:09:16.033+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_stage permission to 509
[2023-06-27T09:09:16.034+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_stage permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_stage'
[2023-06-27T09:09:16.037+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Setup.create_stage manual__2023-06-27T09:09:11.133710+00:00 [running]> on host 4d37da440d5c
[2023-06-27T09:09:16.191+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Setup' AIRFLOW_CTX_TASK_ID='create_stage' AIRFLOW_CTX_EXECUTION_DATE='2023-06-27T09:09:11.133710+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-27T09:09:11.133710+00:00'
[2023-06-27T09:09:16.210+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-27T09:09:16.211+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run-operation create_stage --args \'{                            "stage_name": "uni_kishore_pipeline",                             "stage_url": "s3://uni-kishore-pipeline",                             "dry_run": false}\'']
[2023-06-27T09:09:16.224+0000] {subprocess.py:86} INFO - Output:
[2023-06-27T09:09:19.887+0000] {subprocess.py:93} INFO - [0m09:09:19  Running with dbt=1.5.1
[2023-06-27T09:09:27.329+0000] {subprocess.py:93} INFO - [0m09:09:27  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-27T09:09:27.330+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2023-06-27T09:09:27.331+0000] {subprocess.py:93} INFO - - models.badge_5.create_stage
[2023-06-27T09:09:27.331+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day
[2023-06-27T09:09:27.408+0000] {subprocess.py:93} INFO - [0m09:09:27  Found 2 models, 0 tests, 0 snapshots, 0 analyses, 455 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-27T09:09:28.688+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-27T09:09:28.715+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Setup, task_id=create_stage, execution_date=20230627T090911, start_date=20230627T090915, end_date=20230627T090928
[2023-06-27T09:09:28.790+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-27T09:09:28.820+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
